app/Components/NotificationBanner.php

```diff
@@ -63,9 +63,11 @@ public function displayNotificationBanner(): void
 <?= esc_html($heading); ?>
       </h2>
     </div>
+<?php if (!empty($content)): ?>
```

Comment:
Should we trim the content first in case there's some whitespace?

Reply:
This is a good point. A longer term solution would be for the options page to refuse to submit the form if the heading or content is empty, but I think we need both that and your change, and it's fine to put form validation on a backlog somewhere to be done outside of the DHSC project